packages/ansible-mcp-server/src/server.ts

```diff
@@ -74,15 +74,21 @@ export function createAnsibleMcpServer(workspaceRoot: string) {
     {
       title: "Ansible Lint",
       description:
```

Comment:
For the ansible_lint tool, we should have additional definitions like what we have for ADE and etc here: https://github.com/ansible/vscode-ansible/pull/2219/files#diff-9c2f34b820f924878e19c25de4405aa87443ae392c5c0cfcf9e3b39467816d32R70
Annotations, keywords, etc.